.storybook/themes/news.scss

```diff
@@ -1183,30 +1183,6 @@
 						),
 					),
 				),
-				large-manual-promo: (
```

Comment:
large-manual-promo should not be getting removed in this pr. Not sure I understand -- maybe bad merge?